### PR TITLE
Add general vulnerability findings worfklows

### PR DESCRIPTION
**Pull request by christophpre-dynatrace**
This PR adds workflows to send reports to jira/slack for new general critical vulnerabilities found in the last 24hrs.

### CODE REVIEW

File: samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+          {% for component in _.finding.vulnerability_finding_events  %}
+
+          |{{ component.get("vulnerable_component.short_name") }}|{{
+          component.get("vulnerable_component.version") }}|
+
+          {% endfor %}
+        executionId: "{{ execution().id }}"
+        connectionId: ""
+        fieldSetters: []
+        environmentUrl: https://umsaywsjuo.dev.apps.dynatracelabs.com
```
Comment (agrimmer) [inline samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml:138]: Please don't hardcode environments but use expressions.

File: samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    create_task_in_jira:
+      name: create_task_in_jira
+      description: Create a Jira ticket for the vulnerability findings.
+      action: dynatrace.jira:jira-create-issue
+      active: true
+      input:
+        labels: []
+        taskId: "{{ task().id }}"
+        project:
+          id: "19691"
```
Comment (agrimmer) [inline samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml:78]: The project id is specific to the Jira instance.

Comment (christophpre-dynatrace) [reply samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml]: Thank you,  removed the projectId

File: samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml (hunk 1 above)
Comment (christophpre-dynatrace) [reply samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml]: Thank you, used an expression for the environmentUrl

File: samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml [hunk 3]
```diff
@@ -0,0 +1,163 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.automations
+        version: ^1.728.0
```
Comment (wlfghdr) [inline samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml:6]: version is not released, but at least everything seems to work after importing the WF in prod

File: samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+      - id: dynatrace.jira
+        version: ^4.1.1
```
Comment (wlfghdr) [inline samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml]: same: version is not released, but at least everything seems to work after importing the WF in prod

File: samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml [hunk 5]
```diff
@@ -0,0 +1,232 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.automations
+        version: ^1.728.0
```
Comment (wlfghdr) [inline samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml:6]: same: version is not released, but at least everything seems to work after importing the WF in prod

File: samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+      - id: dynatrace.slack
+        version: ^2.0.2
```
Comment (wlfghdr) [inline samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml]: same: version is not released, but at least everything seems to work after importing the WF in prod

File: samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml (hunk 3 above)
Comment (wlfghdr) [reply samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml:6]: ah - is released - just not deployed in demo.live - all good

File: samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml (hunk 5 above)
Comment (wlfghdr) [reply samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml:6]: ah - is released - just not deployed in demo.live - all good

**Review by wlfghdr — APPROVED** [review]
changed jira and slack versions to released versions and tested import & further editing (did not test execution)

File: samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+        y: 2
+      predecessors:
+        - get_vulnerabilites
+      conditions:
+        states:
+          get_vulnerabilites: SUCCESS
+        custom: '{{result("get_vulnerabilites").records|length > 0}}'
+      concurrency: 1
+      withItems: finding in {{ result("get_vulnerabilites").records }}
+    get_vulnerabilites:
```
Comment (amohrd) [inline samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml:167]: @christophpre-dynatrace I tried to execute the workflow in umsay, the DQL stopped with an error after 5:55h. Do you see any change to optimize the query or limit it to maybe 2h, or provide some hints around scope and execution time?

**Comment by amohrd** [comment]
E2E test successful on umsay

**Review by amohrd — APPROVED** [review]
E2E test on umsay successfully executed.

**Pull request merged**